### DISCUSSION TITLE
Constraining output within some range

### DISCUSSION THREAD

**Discussion by nirmal129** [lululxvi/deepxde#1355] [Q&A]
Hello, 

I am working on a PINN project where the true solution lies in [-1, 1]. However, PINN without any output transformation returns a solution that goes till y=2. I tried to constrain the output by using a transformation y=tanh(y) as the range of tanh is [-1, 1]. This improved my solution but I am still getting errors. I am positive that this error might be because of the way I used the transformation. Is there any other efficient transformation to contain the output in the range [-1, 1]? 

Thank you!

**Comment by lululxvi** [comment]
y=tanh(y) is a right way.
